teams/migration/membership.json

```diff
@@ -48,8 +48,8 @@
 
    "repositories": [
         "dm",
-        "dumpling",
-        "ticdc",
-        "tidb-binlog"
```

Comment:
let's keep "dumpling" in the list, it is still used when cherry-picking to versions before merging into tidb

Reply:
I've consulted @Mini256 , this list is currently not used by anything. So removing it will not affect anything. I prefer to discard it now in case nobody cares it in the future.